src/einsteinpy/tests/test_metric/test_kerrnewman.py

```diff
@@ -70,22 +71,21 @@ def test_calculate_trajectory1(M, r, end_lambda, stepsize):
     assert_allclose(ans[1][0][1], ans[1][-1][1], 1e-2)
 
 
-@pytest.mark.parametrize(
-    "pos_vec, vel_vec, q, M, a, Q, el, ss",
-    [
-        (
-            [1000.0 * u.km, 0.6 * np.pi * u.rad, np.pi / 8 * u.rad],
-            [10000 * u.m / u.s, -0.01 * u.rad / u.s, 0.0 * u.rad / u.s],
-            1 * u.C / u.g,
-            0.5 * 5.972e24 * u.kg,
-            1e-6,
-            100 * u.C,
-            200.0,
-            1.0,
-        )
-    ],
-)
-def test_compare_calculate_trajectory_iterator_bl(pos_vec, vel_vec, q, M, a, Q, el, ss):
+@pytest.fixture()
+def test_input():
+    q = 1 * u.C / u.g
+    a = 1e-6
+    Q = 100 * u.C
+    el = 200.0
+    ss = 1.0
+    return (q, a, Q, el, ss)
```

Comment:
```suggestion
    return q, a, Q, el, ss
```
This will also work :)